ViewUser.js

```diff
@@ -562,8 +562,20 @@ class ViewUser extends React.Component {
         </IfPermission>
 
         <IfPermission perm="circulation.loans.collection.get">
-          <IfInterface name="circulation" version="4.0">
-            <IfInterface name="loan-policy-storage" version="1.0">
+          <IfInterface name="loan-policy-storage" version="1.0">
+            { /* Separate checks for each version of interface, consolidate if possible */ }
+            <IfInterface name="circulation" version="3.0">
```

Comment:
This duplication is vile. Obviously it would be much better to upgrade `<IfInterface>` so it can accept a space-separated list of interface versions, like the `okapiInterfaces` specification does in the package file.

What's the urgency of this? I can add that facility today. But if you need to get this done soonest, then just merge it as it is, and I will live with the bitter, incessant burning deep within my soul at having been complicit in birthing this abomination.

Reply:
I agree, I dislike the duplication as well, this was purely for expediency :-( 

I couldn't see another way to do this (my react isn't good enough to know if there is a way to refer to the same component instance twice without duplication).

A facility to express both expressions in a single expression would be great. (I tried to build stripes-components locally, at the weekend to see if I could do this, but I could get the tests to run without locking up).

The urgency is only that the folio-snapshot build will not be able to form a cohesive backend until 3.x is supported again.

An alternative (as I wondered about in my pull request notes), is to use the check that does not specify a version? 

That might be temporarily preferable to the duplication, until the facility to check for either is available?